context/src/main/java/io/opentelemetry/context/propagation/MultiTextMapPropagator.java

```diff
@@ -16,17 +16,29 @@
 import javax.annotation.Nullable;
 
 final class MultiTextMapPropagator implements TextMapPropagator {
```

Comment:
I think it's appropriate to rename this to `CompositeTextMapPropagator` in this PR